pkgs/sdk/vm/keeper.go

```diff
@@ -406,6 +406,9 @@ func (vm *VMKeeper) QueryFile(ctx sdk.Context, filepath string) (res string, err
 	dirpath, filename := std.SplitFilepath(filepath)
 	if filename != "" {
 		memFile := store.GetMemFile(dirpath, filename)
+		if memFile == nil {
+			return "", fmt.Errorf("file %s is not available", filepath)
```

Comment:
```suggestion
			return "", fmt.Errorf("file %q is not available", filepath) // TODO: XSS protection
```

It will quotes the string + comment about taking care of XSS in the future